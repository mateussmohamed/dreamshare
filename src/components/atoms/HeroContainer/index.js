import styled from 'styled-components'
import { Flex } from '@rebass/grid'

const HeroContainer = styled(Flex)`
  width: 100%;
  height: 100vh;
  flex-direction: column;
`

export default HeroContainer
