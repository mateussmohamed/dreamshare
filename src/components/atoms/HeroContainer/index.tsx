import styled from 'styled-components'
import { Flex } from '@rebass/grid'

export const HeroContainer = styled(Flex)`
  width: 100%;
  height: 100vh;
  flex-direction: column;
`
