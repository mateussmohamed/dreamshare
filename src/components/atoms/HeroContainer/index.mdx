---
name: HeroContainer
menu: Atoms
---

import { Playground, PropsTable } from 'docz'
import HeroContainer from './index'

# HeroContainer
<PropsTable of={HeroContainer} />

<Playground>
  <HeroContainer>
    SHARE YOUR HOLIDAY DREAM
  </HeroContainer>
</Playground>
