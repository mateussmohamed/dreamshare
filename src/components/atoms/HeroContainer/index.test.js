import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import HeroContainer from './index'

describe('[<HeroContainer />]', () => {
  const wrap = (props = {}) => shallow(<HeroContainer {...props} />)

  test('matches to snapshot', () => {
    const wrapper = wrap({ children: 'I am a container!' })

    expect(wrapper).toMatchSnapshot()
  })
})
