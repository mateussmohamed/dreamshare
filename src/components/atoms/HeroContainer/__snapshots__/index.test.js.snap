// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<HeroContainer />] matches to snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "HeroContainer-mwsivc-0",
        "isStatic": false,
        "rules": Array [
          "box-sizing: border-box;",
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          "display: flex;",
          [Function],
          [Function],
          [Function],
          [Function],
          "width:100%;height:100vh;flex-direction:column;",
        ],
      },
      "displayName": "HeroContainer",
      "propTypes": Object {
        "alignItems": [Function],
        "flexDirection": [Function],
        "flexWrap": [Function],
        "justifyContent": [Function],
      },
      "render": [Function],
      "styledComponentId": "HeroContainer-mwsivc-0",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  I am a container!
</StyledComponent>
`;
